client/signup/steps/design-type-with-store/index.jsx

```diff
@@ -85,16 +95,35 @@ class DesignTypeWithStoreStep extends Component {
 	};
 
 	renderChoice = ( choice ) => {
-		return (
-			<Card className="design-type-with-store__choice" key={ choice.type }>
-				<a className="design-type-with-store__choice-link"
-					href="#"
-					onClick={ this.handleChoiceClick( choice.type ) }>
-					{ choice.image }
-					<h2>{ choice.label }</h2>
-				</a>
-			</Card>
-		);
+		switch ( abtest( 'signupCopy' ) ) {
+			case 'newSignupCopy':
+				return (
```

Comment:
I wasn't sure how to handle conditions within in html block being rendered in react. My worry with leaving choice.description blank for the control is then the wrapper markup would still show.